lib/utils.coffee

```diff
@@ -34,4 +34,4 @@ THE SOFTWARE.
 # channel = utils.getChannel('...')
 ###
 exports.getChannel = (device) ->
-	return device.logs_channel or "device-#{device.uuid}-logs"
+	return "device-#{device.logs_channel or device.uuid}-logs"
```

Comment:
@emirotin I don't since I didn't find a use case for it from the context of the CLI. I'll implement it once we port the SDK to the browser so there is a client using it to prevent it from getting rot.